src/components/slider/slider.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link
+			href="https://unpkg.com/boxicons@2.1.1/css/boxicons.min.css"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="/style.css" />
+		<title>Slider</title>
+	</head>
+
+	<body>
+		<body>
+			<header class="header flex">
+				<div class="brand__text grid-center">Magnificent UI</div>
+				<div class="social__handles">
+					<a class="social__links theme__toggler">
+						<i class="bx bxs-moon"></i>
+					</a>
+					<a href="#" class="social__links">
+						<i class="bx bxl-twitter"></i>
+					</a>
+					<a href="#" class="social__links">
+						<i class="bx bxl-linkedin-square"></i>
+					</a>
+					<a href="" class="social__links">
+						<i class="bx bxl-github"></i>
+					</a>
+				</div>
+			</header>
+			<nav class="nav flex-col-align-center">
+				<h1 class="h4 nav__head">Get Started</h1>
+				<ul class="nav__items flex-col-align-center">
+					<li class="nav__list">
+						<a class="nav__links" href="/src/components/usage/usage.html">Usage</a>
+					</li>
+					
+					<li class="nav__list"><a class="nav__links" href="/src/components/typography/typography.html">Typography</a></li>
+				</ul>
+				<h1 class="h4 nav__head">Components</h1>
+				<ul class="nav__items flex-col-align-center">
+					<li class="nav__list">
+						<a
+							class="nav__links"
+							href="/src/components/avatar/avatar.html"
+							>Avatar</a
+						>
+					</li>
+					<li class="nav__list">
```

Comment:
work on indentation

Reply:
Will look into it. Thanks for the review.